crates/bevy_window/src/window.rs

```diff
@@ -534,6 +534,7 @@ pub struct WindowDescriptor {
     pub cursor_visible: bool,
     pub cursor_locked: bool,
     pub mode: WindowMode,
+    pub transparent: bool,
```

Comment:
I would like to document the cross-platform limitations here as well, to improve visibility.

IMO this should be the canonical place for it,  and the example should have a brief link to these docs.